# PeerJ91495
Code for PeerJ 91495 publication

#Reiteration: We strongly recommend refering to respective manuals and vignette to understand the principles and parameters of these programs. 
#Original authors/ developers retain the original and supplementary rights. 
#Code for PeerJ Article 91495 (https://peerj.com/manuscripts/91495/)
#Genomic landscape reveals evolutionary footprints of adaptability in Indian hill cattle 
#The codes used in R-programming environment and other programs (PLINK, etc.) have been delineated in two files for easy follow by readers.


The code on Github provided section wise code for the publication entitled "Genomic landscape reveals evolutionary footprints of adaptability in Indian hill cattle" under active consideration at PeerJ journal platform. 
The sections and code under the manuscript include:
#2.2 Preparation of datasets, quality control, and phasing 
#2.3 Population structure 
#2.4 Selection signature analysis 
#2.4.1 Intra-population selection signatures
#2.4.2 Inter-population selection signatures
#2.5 Functional annotation of selection signature regions


The programs used include
Software/ programs/ tools used in the analysis of data and visualization of results
S. No.	Software/ Program	  Purpose*	    Link to Manual/Vignette
1.		VcfTools 	  Merging of different datasets	https://vcftools.github.io/index.html 

2.		PLINK v1.9	Quality control of genotyping data 	https://zzz.bwh.harvard.edu/plink/ 

3.		TreeMix v1.13	Assessing population structure by inferring the patterns of population splits and mixtures 	https://bitbucket.org/nygcresearch/treemix/wiki/Home 

4.		GCTA v1.93	Generating genomic relationship matrix and principal component analysis	https://yanglab.westlake.edu.cn/software/gcta/ 

5.		REHH R package v3.2.2 	Assessing haplotype statistics: within population (iHS) and cross population (XP-EHH)	https://cran.r-project.org/web/packages/rehh/vignettes/rehh.html 

6.		ggplot2 R package v3.4.4	Visualization of results and plotting Manhattan plots 	https://cran.r-project.org/package=ggplot2 

7.		DAVID	Provide functional information on gene list and underlying pathways	https://david.ncifcrf.gov/summary.jsp 

8.		BioMart ENSEMBL	Identification of overlapping genes around selective sweeps	https://asia.ensembl.org/info/data/biomart/index.html 

*Purpose is as per usage in the current manuscript. The software/ programs/ packages may be useful for other purposes too. 

#For any queries, contact corresponding author at firdousa61@gmail.com
